dev/lib/product_taxonomy/models/integration_version.rb

```diff
@@ -98,17 +129,22 @@ def generate_distribution(output_path:, direction:)
     # Generate a JSON representation of the integration version for a single direction.
     #
     # @param direction [Symbol] The direction of the distribution file to generate (:from_shopify or :to_shopify).
-    # @return [Hash]
+    # @return [Hash, Array<Hash>]
     def to_json(direction:)
+      if direction == :both
+        return [to_json(direction: :from_shopify), to_json(direction: :to_shopify)].compact
+      end
+
+      return @to_json[direction] if @to_json.key?(direction)
```

Comment:
I feel like it'd be better to just embrace the `if`'s of this method (channelling my inner sandi metz)